### DISCUSSION TITLE
Bottom bar way to big

### DISCUSSION THREAD

**Discussion by Charkel** [anapnoe/stable-diffusion-webui-ux#59] [Ideas]
The bottom "bar" is taking up way to much space on my 1920x1080 PC display. 
Top bar and it's buttons wouldn't hurt to loose about 25% size too.

**Comment by prog0111** [comment]
Completely agree.  There may be a CSS hack to get rid of the bottom bar and shrink the top if anyone knows how

**Comment by 779575267** [comment]
agree!!
